src/widgets/DataTable/HeaderCell.js

```diff
@@ -9,15 +9,18 @@ import { getAdjustedStyle } from './utilities';
  *
  * Renders a title and if passed, a sorting icon.
  *
- * @prop {String}       title                 Text to display in the cell
- * @prop {String}       columnKey             The key for the column the cell heads.
- * @prop {Func}         onPressAction         Action for dispatching on press
- * @prop {Func}         dispatch              Dispatcher to backing reducer
- * @prop {ReactElement} SortAscComponent      Icon component for ascending sorting
- * @prop {ReactElement} SortDescComponent     Icon component for descending sorting
- * @prop {ReactElement} SortNeutralComponent  Icon component for neutral state, no sort.
- * @prop {Object}       containerStyle        Style object for the wrapping Touchable or View.
- * @prop {Object}       textStyle             Style object for the inner text component.
+ * @param {String}       title                 Text to display in the cell
+ * @param {String}       columnKey             The key for the column the cell heads.
+ * @param {Func}         onPressAction         Action for dispatching on press
+ * @param {Func}         dispatch              Dispatcher to backing reducer
+ * @param {ReactElement} SortAscComponent      Icon component for ascending sorting
+ * @param {ReactElement} SortDescComponent     Icon component for descending sorting
+ * @param {ReactElement} SortNeutralComponent  Icon component for neutral state, no sort.
+ * @param {Object}       containerStyle        Style object for the wrapping Touchable or View.
+ * @param {Object}       textStyle             Style object for the inner text component.
+ * @param {Number}        width               Optional flex property to inject into styles.
+ * @param {Bool}          isLastCell          Indicator for if this cell is the last
```

Comment:
Once space out 😛

Reply:
I do not have the required skills to fix this :(